### PR TITLE
feat(metadata-service): implement Project Page (DSP-1488)

### PR DESCRIPTION
**Pull request by mpro7**
resolves DSP-1488

I think I'm done with first iteration of another iteration ;) This is stable version of basic layout implementation. Further restyling and adjustments will be done in another task, because this PR is already grown big. I've added also you @danielasubotic just to have a look on the final version. Let me know if there is anything to fix/change.

### CODE REVIEW

**Review by danielasubotic — APPROVED** [review]
looks great! :-)
possible changes for the next iteration/questions:

- if I get back from the project metadata to the projects lists, I would like to get back to that page where the project card was (e.g., project card was on page 2, I would like to get back on page 2 after looking at the project metadata)
- when I search for a project in the full-text search, it is correctly displayed. Does it make sense to have a function/button? to see all projects again? Or we leave it just by clicking on the logo to reset the search?
- Discipline: How to display the data if it is from the SNF-Discipline PDF or from SKOS Unesco?
- Temporal Coverage: How to display the data if there are several entries? 
- I would like to have the full-text search functioning on the project metadata page as well
- How do the tabs on the dataset information look like if there are several datasets with long titles?

**Comment by mpro7** [comment]
Thanks @danielasubotic for having a look. Answering briefly to your questions:
1. This means routing needs to be improved, it's seems it doesn't store the state.
2. We can thing about it.
3. Good question.
4. This and displaying some other properties we need to rethink.
5. Yes, this isn't enabled there yet, but it's also related to 1.
6. They are displayed one under another and active one has always different background. I have few ideas how to change it if we don't like it.

We will talk about the details un the meeting.

File: services/metadata/backend/data/db.json [hunk 1]
```diff
@@ -505,7 +505,7 @@
                         {
                             "place": {
                                 "name": "Geonames",
-                                "url": "https://www.geonames.org/2661604"
+                                "url": "https://www.geonames.org/2661604/basel.html"
```
Comment (BalduinLandolt) [inline services/metadata/backend/data/db.json:508]: did you add those manually? I'm not sure we can guarantee too have the proper name of the place all the time

File: services/metadata/frontend/project-page/DefaultTabComponent.svelte [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    .grid-wrapper {
+      grid-template-columns: repeat(2, 1fr);
+    }
+  }
+  @media screen and (min-width: 768px) {
+    .grid-wrapper {
+      grid-template-columns: repeat(3, 1fr);
+    }
+  }
+</style>
```
Comment (BalduinLandolt) [inline services/metadata/frontend/project-page/DefaultTabComponent.svelte:203]: we should probably at some point decide on how big we want our components to be at max; and when we want to start splitting things up into smaller components.  
(Not saying this is too big, just made me think.)

**Review by BalduinLandolt — APPROVED** [review]
Looks good to me. Details we can discuss in the meeting.

File: services/metadata/backend/data/db.json (hunk 1 above)
Comment (mpro7) [reply services/metadata/backend/data/db.json:508]: Correct, in the case of Geonames it's most probably possible to have it like that, but the other services don't contain anything good to display. This is definitely something we need to talk about.

File: services/metadata/frontend/project-page/DefaultTabComponent.svelte (hunk 2 above)
Comment (mpro7) [reply services/metadata/frontend/project-page/DefaultTabComponent.svelte:203]: I see two main reasons for splitting:
- functional - better reusability + less code redundancy
- structural - if it grows to big, it's harder to maintain and read the code

With the complete design provided upfront it's easier to predict how to divide the app into components. Here w'll need to adapt this strategy in further iterations.

**Pull request merged**